overdrive.py

```diff
@@ -127,11 +128,14 @@ class OverdriveAPI(object):
 
     EVENT_DELAY = datetime.timedelta(minutes=120)
 
-    # The ebook formats we care about.
-    FORMATS = "ebook-epub-open,ebook-epub-adobe,ebook-pdf-adobe,ebook-pdf-open"
+    # The formats we care about.
+    FORMATS = "ebook-epub-open,ebook-epub-adobe,ebook-pdf-adobe,ebook-pdf-open,audiobook-overdrive".split(",")
```

Comment:
The old code worked by accident: we were checking whether a format was a substring of FORMATS rather than that it was in the list FORMATS.